src/components/speakers-page/speakers-list.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { media } from '../../utils/css-utils';
+import BlockHeader from '../common/block-header';
+import MainContainer from '../common/main-container';
+import { FilterBlock, Search } from '../page-filter';
+import SpeakerCard from '../../components/speaker-card/speaker-card';
+
+const SpeakerCardContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+     ${media.phone`
+        flex-flow: row wrap;  
+        justify-content: flex-start;
+    `}
+`;
+
+const LoadMoreButton = styled.button`
```

Comment:
please extract button from subscribtion form or yours one to common component and use it in both places